bin/email-failed-echo-bulky-payment

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname( File::Spec->rel2abs($0) );
+    require "$d/../setenv.pl";
+}
+
+use DateTime;
+use FixMyStreet::Cobrand;
+use FixMyStreet::DB;
+use FixMyStreet::Email;
+use mySociety::Locale;
+
+mySociety::Locale::gettext_domain('FixMyStreet');
+
+my $comment_text = 'Echo failure email sent';
+
+my %cobrands = (
+    kingston => FixMyStreet::Cobrand->get_class_for_moniker('kingston')->new,
+    sutton   => FixMyStreet::Cobrand->get_class_for_moniker('sutton')->new,
+);
+
+my $dtf         = FixMyStreet::DB->schema->storage->datetime_parser;
+my $cutoff_date = $dtf->format_datetime(
+    DateTime->now( time_zone => 'Europe/London' )->subtract( hours => 1 ) );
```

Comment:
Echo sometimes has hours of maintenance downtime, so this could lead to issues during those periods. Perhaps 3 hours should cover it?